src/HttpClientWrapper.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Amazon.Lambda.Core;
+using com.signalfuse.metrics.protobuf;
+using ProtoBuf;
+
+namespace SignalFx.LambdaWrapper
+{
+    public class HttpClientWrapper
+    {
+        private static readonly string DefaultBaseAddressUrl = "https://ingest.signalfx.com";
```

Comment:
default to pops.signalfx.com and make configurable
and make scheme configurable [default https]
add default port and make configurable [default 443]